controllers/suite_test.go

```diff
@@ -92,6 +95,11 @@ func TestAPIs(t *testing.T) {
 			testDeploymentTypeEnvVarName, convergedDeploymentType, providerDeploymentType, consumerDeploymentType))
 	}
 
+	networkType = os.Getenv(testNetworkTypeEnvVarName)
+	if networkType != OVNNetworkType && networkType != SDNNetworkType {
+		panic(fmt.Sprintf("Enviroment var '%s' should be set to one of '%s' or '%s' values", testNetworkTypeEnvVarName, OVNNetworkType, SDNNetworkType))
+	}
+
```

Comment:
We should be able to test even with non-supported values to validate behavior in these cases as well.
Please remove this check.